README.md

```diff
@@ -4,87 +4,92 @@
-
-You may also need **Visual Studio 2022** or later with **WindowsAppSDK 1.2 Stable C# Extension** installed.
-Please refer to this link to download the extension for your Visual Studio version
-- [**Visual Studio 2022 C# Extension**](https://aka.ms/windowsappsdk/1.2/latest/WindowsAppSDK.Cs.Extension.Dev17.Standalone.vsix)
+> Using a lower Visual Studio version (like VS2019) is possible, but you is not recommended as you need to downgrade **WindowsAppSDK** via *NuGet* to **WindowsAppSDK 1.1.5** or **WindowsAppSDK 1.2-preview2** before building. This has an increased risk of breaking the application and as such, minimal support will be provided for this method. **This is not recommended for beginner users.**
 
-# Third-party repositories and references being used in this project
+# Third-party repositories and libraries used in this project
 - [**Windows UI Library**](https://github.com/microsoft/microsoft-ui-xaml) by Microsoft
 - [**HDiffPatch**](https://github.com/sisong/HDiffPatch) by housisong
 - [**Color Thief .NET**](https://github.com/KSemenenko/ColorThief) by KSemenenko
 - [**SevenZipExtractor**](https://github.com/neon-nyan/SevenZipExtractor) by adoconnection
 - [**Newtonsoft.Json**](https://github.com/JamesNK/Newtonsoft.Json) by JamesNK
 - [**Hi3HelperCore.Http**](https://github.com/neon-nyan/Hi3HelperCore.Http) by neon-nyan
 
-**Disclaimer**: This project **IS NOT AFFILIATED** with [**miHoYo (miHoYo Co., Ltd.)**](https://www.mihoyo.com/) or [**HoYoverse (COGNOSPHERE PTE. LTD.)**](https://www.hoyoverse.com/en-us) by any means and completely open-sourced and made with love by Kanchous'. Any contributions are welcomed! 😃
+**Disclaimer**: This project **IS NOT AFFILIATED** with [**miHoYo (miHoYo Co., Ltd.)**](https://www.mihoyo.com/) or [**HoYoverse (COGNOSPHERE PTE. LTD.)**](https://www.hoyoverse.com/en-us) by any means and is completely open-sourced. Any contributions are welcomed! 😃
 
 # UI Design Overview
 ![](https://raw.githubusercontent.com/neon-nyan/CollapseLauncher-Page/main/images/UI%20Overview%20RC2.webp)
 
-# How to support us?
-We are now opening sponsorship through **GitHub Sponsors**, **DANA (Indonesian-only Digital Wallet)**, and **PayPal** to keep this and other projects alive.
-By giving us your support, hope we could keep the server used for the API of this and other projects maintained and make more improvements for this project overtime! :smile:
+# How can I support this project?
+Supporting is never an obligation but is always appreciated and motivates us to invest more time in the project and to keep this and other projects alive. To this end, should you decide to support us, here are ways you can do so :smile::
+- **[GitHub Sponsors](https://github.com/sponsors/neon-nyan)**
+- **[DANA (Indonesian-only Digital Wallet)](https://link.dana.id/qr/bbjthkk)**
```

Comment:
Adding disclaimer that DANA method it currently is not working is probably a good idea. 

@neon-nyan might want to switch to QRIS method instead for easier payment method method as some banks have feature to request money (or in this case, support/donation) this way. You could also seems to be able to get QRIS directly from them https://qris.id/ but this is usually done for business so idk if they allow you to use it for personal use.
Though, if you're not feeling comfortable with it, you can do what ID streamers usually do with [Saweria](https://saweria.co/) or [trakteer](https://trakteer.id/).
Suggesting this because DANA seems to be less and less used at this point in favor for another E-money providers.

Reply:
I'll leave this open for @neon-nyan but I added a sentence clarifying that DANA is currently unsupported.

Reply:
Thanks for the suggestion. I'm planning to move to QRIS. For now, we can just remove DANA from the line.

Reply:
Sounds, good. If it looks good to know on the latest commit, you can go ahead and merge this one.